build.sbt

```diff
@@ -67,15 +67,24 @@ def isolateAllTests(tests: Seq[TestDefinition]) = tests map { test =>
   } toSeq
 
 // Subproject definitions begin
-
-// NB: FIRRTL dependency is unmanaged (and dropped in sim/lib)
+//
+// NB: FIRRTL should not be a managed dependency of chisel or rocketchip.
+// Instead, they will get firrtl from the JAR file placed in /lib
 lazy val chisel  = (project in rocketChipDir / "chisel3")
 
+lazy val firrtl = (project in file("tools/firrtl"))
```

Comment:
this shouldn't be required.

Reply:
The problem is probably that these tools are requesting a different version of firrtl than the one we are providing in lib

Reply:
But the FIRRTL jar we are providing in lib was generated from tools/firrtl, wasn't it?

Reply:
What uses tools/firrtl? I would expect the firrtl in rocketchip to get used.

Reply:
Everything should use the FIRRTL in `tools/firrtl`. We want to decouple RC's FIRRTL from the FIRRTL that we use so we could be ahead/behind what they provide.